rllib/connectors/module_to_env/remove_single_ts_time_rank_from_batch.py

```diff
@@ -50,9 +50,21 @@ def __call__(
         if shared_data is None or not shared_data.get("_added_single_ts_time_rank"):
             return data
 
-        data = tree.map_structure_with_path(
-            lambda p, s: s if Columns.STATE_OUT in p else np.squeeze(s, axis=0),
-            data,
-        )
+        def _remove_single_ts(item, eps_id, aid, mid):
```

Comment:
bug fix: For mixed-MultiRLModules where some RLModules are NOT stateful, the old code would crash.